app/components/questions/RideTripsOhare.js

```diff
@@ -0,0 +1,130 @@
+import { useState, useEffect } from 'react'
+import useFetch from 'use-http'
+import { Table } from '../../components/Table'
+import { Notification } from '../../components/Notification'
+import { FailureNotification } from '../../components/Notification'
```

Comment:
You can use `FailureNotification` instead of `Notification` and remove `Notification` entirely.